.github/workflows/release.yml

```diff
@@ -13,43 +13,21 @@ jobs:
     steps:
     - name: git co
       uses: actions/checkout@v1
-
+    - name: Get Tag Info
+      id: release-tag-meta
+      uses: ./.github/actions/release-tag-meta
+      with:
+        git-ref: ${{ github.ref }}
+    - run: echo ${{ toJSON(steps) }}
     - name: make package
       env:
-        CARGO_RELEASE: 1
-      run: PACKAGE_VERSION=$(echo $GITHUB_REF | sed -E 's,^refs/tags/release/,,') make package
-
-    - uses: actions/upload-artifact@v1
-      with:
-        name: package
-        path: target/release/package
-
-  test:
-    runs-on: ubuntu-latest
-    container:
-      image: docker://rust:1.37-buster
-    steps:
-    - name: git co
-      uses: actions/checkout@v1
-    - run: make test-integration
-
-  publish:
-    needs: [package, test]
-    runs-on: ubuntu-latest
-    steps:
-    - name: fetch
-      uses: actions/download-artifact@v1
-      with:
-        name: package
-        path: package
+        CARGO_RELEASE: "1"
+        PACKAGE_VERSION: ${{ steps.release-tag-meta.name }}
+      run: make package
```

Comment:
how about:
```yaml
run: PACKAGE_VERSION=$(echo "$GITHUB_REF" | sed s,^refs/tags/release/,,) make package
```
... and then forego `Dockerfile`, `entrypoint.sh`, and `action.yml`? i could see utility in the custom action setup if it was doing more stuff.

Reply:
AFAIU: the action is needed so that i can define task outputs that can be referenced for the `name` input of the release action... i don't know of any other way of getting a modified value into that input...